src/test/regress/expected/failure_on_create_subscription.out

```diff
@@ -54,7 +54,7 @@ SELECT citus.mitmproxy('conn.onQuery(query="ALTER SUBSCRIPTION").kill()');
 (1 row)
 
 SELECT master_move_shard_placement(101, 'localhost', :worker_1_port, 'localhost', :worker_2_proxy_port);
-ERROR:  connection not open
+ERROR:  connection pointer is NULL
```

Comment:
This is the first introduction of this error into our codebase. Based on the name it seems like this is actually a bug.